e2e-tests/functions

```diff
@@ -23,19 +23,35 @@ deploy_client() {
 	kubectl -n "${NAMESPACE}" apply -f "${TESTS_CONFIG_DIR}/client.yaml"
 }
 
+deploy_pmm_server() {
+	local platform=kubernetes
+	if [ -n "${OPENSHIFT}" ]; then
+		platform=openshift
+	fi
+
+	helm install monitoring \
+		-n "${NAMESPACE}" \
+		--set platform="${platform}" \
+		"https://percona-charts.storage.googleapis.com/pmm-server-${PMM_SERVER_VERSION}.tgz"
+}
+
 get_cr() {
 	local test_name=$(basename "$(pwd)")
 	yq eval "$(printf '.metadata.name="%s"' "${test_name}")" "${DEPLOY_DIR}/cr.yaml" \
+		| yq eval '.spec.secretsName="test-secrets"' - \
+		| yq eval '.spec.sslSecretName="test-ssl"' - \
 		| yq eval "$(printf '.spec.crVersion="%s"' "${CR_VERSION}")" - \
 		| yq eval "$(printf '.spec.mysql.image="%s"' "${IMAGE_MYSQL}")" - \
-		| yq eval "$(printf '.spec.orchestrator.image="%s"' "${IMAGE_ORCHESTRATOR}")" -
+		| yq eval "$(printf '.spec.orchestrator.image="%s"' "${IMAGE_ORCHESTRATOR}")" - \
+		| yq eval "$(printf '.spec.pmm.image="%s"' "${IMAGE_MYSQL}")" -
 }
 
 run_mysql() {
 	local command="$1"
 	local uri="$2"
+	local pod="$3"
 
-	kubectl -n "${NAMESPACE}" exec mysql-client -- \
+	kubectl -n "${NAMESPACE}" exec "${pod:-mysql-client}" -- \
```

Comment:
just for the sake of uniformity it's better to move variable defaults to variable declaration